flask_atlassian_connect/base.py

```diff
@@ -10,7 +10,7 @@
 
 try:
     # python2
-    from urllib import urlencode
+    from urllib.parse import urlencode
```

Comment:
This doesn't seem right. python3 has urllib.parse

Reply:
hrm, 2to3 did that automatically and it seemed to work. Upon further inspection it looks like urlencode did some very strong reorganization between python 2 and 3:

>urllib is the hardest module to use from Python 2/3 compatible code.
>http://python-future.org/compatible_idioms.html#urllib-module

```
# Python 2 only:
from urllib import urlencode

# Python 3 only:
from urllib.parse import urlencode

# Python 2 and 3: easiest option
from future.standard_library import install_aliases
install_aliases()

from urllib.parse import urlparse, urlencode
```

Reply:
oh interesting. Yea the previous code already supported both by importing one, failing, and importing the other.

I'll look into it a bunch more later, i just noticed that one right away.